components/d2l-activity-editor/lang/es-es.js

```diff
@@ -186,38 +186,38 @@ export default {
 	"content.lastEdited": "Última edición", // The date when the entity was last edited
 	"content.externalFile": "Archivo externo", // The text to use for labeling external files
 	"content.advancedEditing": 'Edición avanzada', // The text to use on the advanced editing button
-	"content.advancedEditingDisabledTooltip": "", // The tooltip of the icon for when advanced editing is disabled
+	"content.advancedEditingDisabledTooltip": "La edición avanzada solo está disponible para el propietario del vídeo. Si desea obtener acceso, póngase en contacto con su administrador.", // The tooltip of the icon for when advanced editing is disabled
 	"content.fileHasCaptions": "Este archivo tiene subtítulos en", // The text to use preceding the list of captions
-	"content.lorRepositoryName": "Repository", // The label for the name of the LOR repository
-	"content.lorSettings": "Settings", // The label to put in the LOR settings button
+	"content.lorRepositoryName": "Repositorio", // The label for the name of the LOR repository
+	"content.lorSettings": "Configuración", // The label to put in the LOR settings button
 
-	"evaluators.lblEvaluators": "Evaluators", //Text label for the evaluators
-	"evaluators.btnManageEvaluators": "Add/edit evaluators", //Button text to launch evaluators management dialog
-	"evaluators.txtNoEvaluatorsAdded": "No explicit evaluators selected", // Evaluators summary for no evaluators
-	"evaluators.txtEvaluatorsAdded": "{count} {count, plural, =1 {evaluator} other {evaluators}} added", // Evaluators summary
-	"evaluators.btnSave": "Guardar", // Save evaluators button
+	"evaluators.lblEvaluators": "Evaluadores", // Text label for the evaluators
+	"evaluators.btnManageEvaluators": "Añadir/editar evaluadores", // Button text to launch evaluators management dialog
+	"evaluators.txtNoEvaluatorsAdded": "No se han seleccionado evaluadores explícitos", // Evaluators summary for no evaluators
+	"evaluators.txtEvaluatorsAdded": "{count} {count, plural, =1 {evaluator} otros {evaluators}} añadidos", // Evaluators summary
```

Comment:
```suggestion
	"evaluators.txtEvaluatorsAdded": "{count} {count, plural, =1 {evaluator} other {evaluators}} añadidos", // Evaluators summary
```

"other" should not be translated